docs/range-tokens/deployment-tutorial.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Range Tokens Deployment Tutorial
+sidebar_label: Deployment Tutorial
+---
+
+The process to launch and manage a range token on UMA is surprisingly simple. This tutorial will show you how to deploy a range token using the Long Short Pair (LSP) contract and the UMA `launch-lsp` repository. By following these instructions, you will be able to launch a custom LSP on Kovan testnet and Ethereum mainnet.
+
+## Deployment process
+
+The following is an overview of the deployment steps below.
+
+1. Clone the repo
+2. Install the dependencies
+3. Set your parameters
+4. Run the deployment on testnet (Kovan)
+5. Run the deployment on mainnet (Ethereum)
+
+### Is your desired price identifier and collateral type supported?
+
+Before deploying a contract, you should verify that your desired price identifier and collateral currency are already approved on the network you are trying to deploy to.
+
+View approved price identifiers and currencies here:
+- [Approved mainnet collateral currencies](/uma-tokenholders/approved-collateral-currencies)
+- [Approved mainnet price identifiers](/uma-tokenholders/approved-price-identifiers)
+- [Approved Kovan collateral currencies subgraph query](https://thegraph.com/explorer/subgraph/umaprotocol/uma-kovan?query=Whitelisted%20Collateral%20Currencies)
+- [Approved Kovan price identifiers subgraph query](https://thegraph.com/explorer/subgraph/umaprotocol/uma-kovan?query=Pricefeed%20Identifiers)
+
+To add a new price identifier or collateral currency to mainnet, please propose one to UMA voters by following the instructions detailed [here](/uma-tokenholders/adding-price-id).
+
+To add a new price identifier or collateral currency to Kovan, please contact the UMA team in Discord.
+
+## Clone the repo
```

Comment:
What do you think about adding a separate launch-lsp prereqs page, since this content will be redundant in all of the deployment tutorials?

Reply:
I agree and I implemented the change we talked about with creating a separate section in Developers > Quick Start > LSP Prerequisites